### PR TITLE
Chromincally typo 5660

### PR DESCRIPTION
**Pull request by anuroop15**
#### Which User story does this relate to?
- https://osi-cwds.atlassian.net/browse/CALS-5660

#### Brief feature description
- 

#### Does this PR block/effect other stories?
- 

#### Do any commands needs to be run in order to run the project with this PR?
- 

#### Code coverage percent diff as a result of this PR?
-

### CODE REVIEW

**Comment by nasar-cambria** [comment]
@anuroop15  Still need to resolve conflicts.

File: app/javascript/constants/rfaText.js [hunk 1]
```diff
@@ -52,6 +52,13 @@ const yesNoMarginStyle = Object.freeze({
   marginBottom: '20px'
 })
 
+const rfa01ALabels = Object.freeze({
+  'previouslyLicensedLabel': 'Have you previously been licensed to operate a non-foster care community care facility, child care center, family child care\n' +
+  'home, or residential care facility for the elderly or chronically ill?',
+  'previouslyEmployedLabel': 'Have you previously been employed by or volunteered at a community care facility, child care center, family child care\n' +
+  'home, or residential care facility for the elderly or chronincally ill?'
```
Comment (nasar-cambria) [inline app/javascript/constants/rfaText.js]: Its still not correct. It should be ```chronically```

**Comment by ratneshraval** [comment]
@nasar-cambria @anuroop15 Should've fixed the typo in the title before merging this in.
Also @anuroop15 all PRs should have all of cals front-end as reviewers.

**Comment by ratneshraval** [comment]
Also PR title doesn't have proper story number.

**Pull request merged**